install_config/install/prerequisites.adoc

```diff
@@ -797,6 +797,45 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+If your container's log (*_/var/lib/docker/containers/HASH/HASH-json.log_*) is
+increasing, you can use `json-file` logging driver options to manage this.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
+`--log-opt max-file`, which is the maximum number of files that a log is rolled
+over before being discarded. You can use these options with Docker 1.8.2 or
+later.
+
+Setting in *_/etc/sysconfig/docker (max-size=1M, max-file=3)_* and restart docker
```

Comment:
Should the file name markup end where the file name ends? Should those command operations be wrapped in back-ticks? Example:

Setting in *_/etc/sysconfig/docker_* (`max-size=1M`, `max-file=3`) and restart docker service:

Also, this description feels awkward somehow by starting with "Setting". And should you include the command which will restart the docker service? I'd consider rephrasing to something like:

Edit the *_/etc/sysconfig/docker_* file to configure `max-size=1M` and `max-file=3`:

====
----
OPTIONS='--insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3' <1>
----
====

Next, restart the docker service:

----
# systemctl restart docker.service
----